lib/tests/formulae.rb

```diff
@@ -64,6 +64,27 @@ def diff_formulae(start_revision, end_revision, path, filter)
         end.compact
       end
 
+      def github_actions_revisions
```

Comment:
Can you line this? The indirection makes it harder to follow when it's only called once and it's harder to see what the changes you've made are.

Reply:
It was about too many if blocks. I perhaps rushed it a little - I'll think of something better.